docker/docker-compose.ha.yml

```diff
@@ -23,7 +23,7 @@ services:
       POSTGRESQL_REPLICATION_MODE: master
       POSTGRESQL_REPLICATION_USER: myreplicationuser
       POSTGRESQL_REPLICATION_PASSWORD: myreplicationpassword
-      PARADEDB_TELEMETRY: true # Set to 'false' (or remove) to disable anonymous telemetry
+      TELEMETRY: true # Set to 'false' (or remove) to disable anonymous telemetry
```

Comment:
I am not convinced this won't cause issues with the Bitnami Dockerfile, have you tested?

Reply:
I haven't, what kind of issues are you worried about? We renamed this variable at some point and just forgot it in this file.

Reply:
Just to be safe, I've made everything `PARADEDB_TELEMETRY` everywhere.